recipes/openssl/1.x.x/conanfile.py

```diff
@@ -757,6 +757,9 @@ def build(self):
                 xcrun = tools.XCRun(self.settings)
                 env_vars["CROSS_SDK"] = os.path.basename(xcrun.sdk_path)
                 env_vars["CROSS_TOP"] = os.path.dirname(os.path.dirname(xcrun.sdk_path))
+                if self.settings.os in ["iOS", "tvOS", "watchOS"] and self.conf.get("tools.apple:enable_bitcode", check_type=bool):
+                    cflags = " ".join(self._get_env_build().vars_dict["CFLAGS"])
+                    env_vars["CFLAGS"] = "%s %s" % (cflags, "-fembed-bitcode")
```

Comment:
🤔 I'd expect the same as boost

https://github.com/conan-io/conan-center-index/blob/e650a185c988ef064029c41e9fd56e79ad4a01ae/recipes/boost/all/conanfile.py#L1116-L1117